rust/foxglove/src/websocket.rs

```diff
@@ -725,6 +737,27 @@ impl Server {
         }
     }
 
+    /// Sets a new session ID and notifies all clients, causing them to reset their state.
+    /// If no session ID is provided, generates a new one based on the current timestamp.
+    pub fn clear_session(&self, new_session_id: Option<String>) {
+        *self.session_id.write() = new_session_id
+            .map(Into::into)
```

Comment:
It's already a String, this line can be deleted

Reply:
oh thanks, AI originally gave me a more complicated Into type for the argument but I simplified it, forgot to remove here